TEveEventDisplay/src/TEveMu2eDataInterface.cc

```diff
@@ -180,15 +206,17 @@ namespace mu2e{
         string pos2D = "(" + to_string((double)hit.pos().x()) + ", " + to_string((double)hit.pos().y()) + ", " + to_string((double)hit.pos().z()) + ")";
```

Comment:
Two comments for future upgrades.  What are sensible precisions - do these give you the default 6 digits.  Is that a good choice?  Write a small free function to format 3-vectors as strings, rather than writing it out every time.